src/rules/prefer-early-return/index.js

```diff
@@ -1,81 +1,81 @@
 /** @type {import('eslint').Rule.RuleModule} */
-module.exports = {
-  meta: {
-    docs: {
-      description:
-        'Prefer early returns over full-body conditional wrapping in function declarations.',
-      category: 'Best Practices',
-      recommended: false,
-      uri: 'https://github.com/cloudfour/eslint-config/blob/main/src/rules/prefer-early-return/README.md',
-    },
-    schema: [
-      {
-        type: 'object',
-        properties: {
-          maximumStatements: {
-            type: 'integer',
-          },
-        },
-        additionalProperties: false,
-      },
-    ],
-  },
+export default {
```

Comment:
This is probably a dumb question but, do we need this file? Can we just import this from somewhere/turn it on somehow? Maybe this one?

https://www.npmjs.com/package/@regru/eslint-plugin-prefer-early-return

Reply:
Maybe? But we already had it in the repo, and for now I'd rather make fewer changes than introduce another dependency from a repo we don't know about. I'll file a followup issue to look into it, though.